usecases/MinimumWorkingExample/perform_calibration_prediction_post_dodo.py

```diff
@@ -0,0 +1,83 @@
+import os
```

Comment:
What is the problem integrating this into the dodo file of the minimum working example? In this approach here, everything is recomputed, even if there are no changes, making the execution of the tasks expensive to work with, even if only small changes not effecting the files are done. And the function is not called in the actions.

Reply:
I tried writing this in dodo, because of its abstraction takes too much of an extra time. I cant debug (atleast easily so dont know what is going on).  I try to rely on as less packages as possible because they come and go. I can include this in the actions. lebedigital.yaml right?